cluster/manifests/spotio-controller/vpa.yaml

```diff
@@ -1,22 +0,0 @@
-{{ if and (index .Cluster.ConfigItems "spotio_account_id") (index .Cluster.ConfigItems "spotio_access_token") }}
```

Comment:
Deletions require an entry in https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/deletions.yaml as well.